data-model/src/main/avro/RawEventLog.avdl

```diff
@@ -0,0 +1,19 @@
+@namespace("org.hypertrace.core.datamodel")
+protocol LogEventRecordsProtocol {
+  import idl "Attributes.avdl";
+
+  record LogEventRecord {
+    string customer_id;
+
+    bytes span_id;
+    bytes trace_id;
+
+    long time_stamp = 0;
```

Comment:
timestamp is usually one word. also indicating the unit here would be great (`timestamp_millis`)